install/kubernetes/helm/meshery-operator/Chart.yaml

```diff
@@ -80,6 +80,10 @@ dependencies:
 - name: meshery-meshsync
   version: 0.1.0
   condition: meshery-meshsync.enabled
+- name: meshery-cilium
+  version: 0.1.0
```

Comment:
We will never keep this version number up to date without automation. Will you confirm that the helm chart releaser will dynamically set this to Meshery's release tag # on release of Meshery?

Reply:
In my opinion, we will manually nump version of the helm charts if we edit the charts manifest. The CI process the keep to bump the `meshery-operator`(the primary chart we always said)  version. But there is not have any logic for subcharts.

Reply:
OK, I see.